include/dlaf/eigensolver/reduction_to_band/impl.h

```diff
@@ -1215,8 +1215,9 @@ Matrix<T, D> ReductionToBand<B, D, T>::call(comm::CommunicatorGrid& grid, Matrix
   Matrix<T, D> mat_taus_retiled =
```

Comment:
Can you comment on the relation of reduction to band changes to potri?

Reply:
#1325
See https://github.com/eth-cscs/DLA-Future/pull/1324#issuecomment-2863479221